redis.conf

```diff
@@ -1440,6 +1440,19 @@ lua-time-limit 5000
 #
 # cluster-migration-barrier 1
 
+# By default when primary becomes empty due to rebalancing, its replicas will
+# reattach to new owner of last migrated slot.
+# Since start of replication is quite cpu consuming operation (because new
+# primary will transmit its dump), this option allows to disable such migration.
+# Also it may help to stabilize cluster topology (together with
+# cluster-migration-barrier set to hight value) in case you rely on external
+# tool for topology management.
+#
+# Default is 'yes' (replica will switch to other master if current master became
+# empty). A value 'no' will disable such automigration.
+#
+# cluster-migrate-from-empty yes
```

Comment:
Yeah, let's rename this to "cluster-allow-replica-migration", it will still default to yes. We can simplify the documentation as well.

The other place to update is in clusterHandleSlaveMigration(), we don't want to migrate. People can use this instead of setting the above cluster-migration-barrier to an arbitrary large value, which should make this experience clearer.

Reply:
I get it. Force pushed version with renaming and handling clusterHandleSlaveMigration.